test/docker-compose.yml

```diff
@@ -451,8 +489,8 @@ services:
       - "-c"
       - "/app/supernets2-node run --network custom --custom-network-file /app/genesis.json --cfg /app/config.toml --components \"rpc,synchronizer\""
 
-  supernets2-data-node-db:
-    container_name: supernets2-data-node-db
+  supernets2-data-availability-db:
+    container_name: supernets2-data-availability-db
```

Comment:
I think in general we should keep our modifications minimum to avoid later merging overhead from the upstream. so this rename seems not necessary